CHANGELOG.rst

```diff
@@ -10,6 +10,13 @@ This document describes changes between each past release.
 
 - Update the Docker compose configuration to use memcache for the cache backend (#1405)
 
+**Operational concerns**
+
+- *The schema for the Postgres ``storage`` backend has changed.* This
+  changes some ID columns to use the "C" collation, which will make
+  delete_all queries faster. This may change the sort order and
```

Comment:
```
``delete_all``
```

Reply:
(suggestion) Maybe a link to a Postgresql reference that mentions why the C collation will leverage the index could be useful